templates/download/desktop/thank-you.html

```diff
@@ -341,14 +346,14 @@ <h3 class="p-card--title">Select topics you&rsquo;re interested in</h3>
     var matchingItem = false;
 
     items.forEach(
-      function(item, itemIndex) {
-        if (
-        item['price'] == totalAmount ||
-        (item['price'] > totalAmount && items[itemIndex - 1]['price'] < totalAmount)
-        ) {
-          matchingItem = items[itemIndex];
-        }
+    function(item, itemIndex) {
+      if (
+      item['price'] == totalAmount ||
+      (item['price'] > totalAmount && items[itemIndex - 1]['price'] < totalAmount)
+      ) {
+        matchingItem = items[itemIndex];
       }
+    }
```

Comment:
Traditionally you put this coding bracket on the same line as the sad face below

Reply:
ok, not my code